src/to_square_example.c

```diff
@@ -0,0 +1,177 @@
+#include <pebble.h>
+
+
+#define ABS(a) (((a) > 0) ? (a) : -1 * (a))
+#define IMAGES_LENGTH 2
+
+
+
+static Window *s_main_window;
+static Layer *icon_layer;
+static GDrawCommandImage* images [IMAGES_LENGTH];
+
+
+static int to_square_normalized;
+static int current_image;
+
+static int16_t prv_int_attract_to(int16_t i, int16_t bounds, int32_t normalized) {
```

Comment:
Also, add a couple of line comments explaining what this part of the code actually does.